### ISSUE TITLE
Error on /foss4g2021/submit/uhklCy/profile/

### ISSUE THREAD

**Issue by ImmaculateMwanja** [pretalx/pretalx#1044]
## Steps to Reproduce

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Chrome 39, python 3.5):
* Operating System and version (desktop or mobile):
* Link to your instance, if in production:

**Comment by vmx** [comment]
Hi @ImmaculateMwanja I guess this was at https://callforpapers.2021.foss4g.org/foss4g2021/ ? 

Could you please be more specific what you the error is about? What did you try to do, what did you expect to happen?

PS: @rixx I'm in touch with the FOSS4G 2021 organisers to set up an issue tracker for their instance. I hope it's OK if I use this issue in the meantime for debugging.

**Comment by rixx** [comment]
Sure, no problem. There's not much I can do without error logs – feel free to update here when/if you know more.

**Comment by ImmaculateMwanja** [comment]
Hi Volker,

Thank you for reaching out in quite a timely manner!

I was submitting my application form for the Call for Proposals for FOSS4G
2021 when I got a notification that there was an error and had to report it
to GitHub. I then re-submitted my proposal and it brought the same error as
before that the submission was unsuccessful, and it prompted me to report
the error to GitHub.

Before submitting for the 3rd time, I went back and looked at my
submissions and found that the proposal was submitted both times it just
did not tell me that my submissions were successful so I had to withdraw
one submission.

I am guessing this may happen to different people and therefore what I
expected to happen is to receive a message that my submission was
successful. I am not sure if this is an error that should be fixed from
your end (pretalx) or by FOSS4G themselves but it seems there is a bug of
some sort.

Very best,
*Immaculata Mwanja*
Operations Manager
OpenMap Development Tanzania <http://omdtz.or.tz/> (HOT Tanzania)
Twitter: imma_mwanja <https://twitter.com/imma_mwanja>
Phone: +255 672 093 027 (Phone and WhatsApp)
Timezone: UTC +03:00 (Dar es Salaam, Tanzania)

Humanitarian OpenStreetMap Team
Using OpenStreetMap for Humanitarian Response & Economic Development
web <http://hotosm.org/> | twitter <https://twitter.com/hotosm> | facebook
<https://www.facebook.com/hotosm> | donate
<http://hotosm.org/donate>

**Comment by gvarela1981** [comment]
Hi @rixx , I'm part of the FOSS4G 2021 volunteer team and I got 2 submitters reporting this situation.
The screenshot we recieved by email is a simple http 500 response in wich it recomends to fill a bug report.
![Untitled](https://user-images.githubusercontent.com/9384999/108561975-e9c0d600-72dd-11eb-9091-fceeeaaa18f5.png)

We do have a few duplicated submissions but mostly submissions seems to be working fine.

Yesterday we where able to perform a successful submission, a simple user story test with the following characteristycs
Track: General Track
Title / Abstract / description: simple ASCII with no special characters
Image: No image
Additional speaker: none
Track / Topic / Level : selected by default
Language: Selected by default
Account: new account
mail: valid mail

Pretalx is running on a dedicated sever, we used a community version implemented using a docker-compose stack.
We are trying different combinations of user stories trying to replicate the http 500 response and will reply all information can get from nginx logs.

Best regads!
